mobile_config/src/client/authorization_client.rs

```diff
@@ -11,7 +11,7 @@ use std::{sync::Arc, time::Duration};
 
 #[async_trait]
 pub trait AuthorizationVerifier: Send + Sync + 'static {
-    type Error;
+    type Error: std::error::Error + Send + Sync + 'static;
```

Comment:
Since these are always `ClientError` can we remove the associated type?

Reply:
removed a223808